upgrade-notes/backend_20250404_220155.md

```diff
@@ -0,0 +1,5 @@
+#### Optimize image resizer with caching ([#3907](https://github.com/shopsys/shopsys/pull/3907))
+
+- Discuss these changes with some SEO expert as they may affect SEO for images
+    - If you don't want to change image behavior for robots just remove `header('X-Robots-Tag: noindex, nofollow');` on line 101 in `app/web/imageResizer.php`
```

Comment:
and in robots.txt.tsx file?

Reply:
So, it should be in the storefront file? If we have this option mentioned, I think it's good to be clear about the need to adjust the storefront and backend simultaneously.

Reply:
I tried to update it a little